src/condor_io/condor_auth_x509.cpp

```diff
@@ -925,7 +948,12 @@ int Condor_Auth_X509::authenticate_client_gss(CondorError* errstack)
             status = 0;
         }
 
-        delete [] server;
+		if (!cred.empty()) {
+			classad::ClassAd ad;
+			ad.InsertAttr("ServerPublicCert", cred);
```

Comment:
The attribute name should be added to condor_attributes.h and the macro used here.